externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -168,15 +169,23 @@ object SparkSQLEngine extends Logging {
 
   def main(args: Array[String]): Unit = {
     SignalRegister.registerLogger(logger)
-    var spark: SparkSession = null
-    try {
-      spark = createSpark()
```

Comment:
we shall break `createSpark` to `setupConf` and ` createSpark` and call `setupConf` before we use `kyuubiConf`

Reply:
> we shall break `createSpark` to `setupConf` and ` createSpark` and call `setupConf` before we use `kyuubiConf`

Yes, i will fix it.